test/__init__.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+
+from betamax import Betamax
+from betamax_serializers.pretty_json import PrettyJSONSerializer
+from requests import Session
+
+if sys.version_info.major == 2:
+    from urllib import quote_plus
+else:
+    from urllib.parse import quote_plus
+
+with Betamax.configure() as config:
+    config.cassette_library_dir = os.path.join('test', 'cassettes')
+    config.serialize_with = 'prettyjson'
+
+    username = os.environ.get('pycaching_test_username') or 'USERNAMEPLACEHOLDER'
+    password = os.environ.get('pycaching_test_password') or 'PASSWORDPLACEHOLDER'
+
+    config.define_cassette_placeholder('<USERNAME>', quote_plus(username))
+    config.define_cassette_placeholder('<PASSWORD>', quote_plus(password))
```

Comment:
I don't know how to achieve this in betamax, but we must also substitute `gspkauth` cookie to prevent [session hijacking](https://www.owasp.org/index.php/Session_hijacking_attack). I verified that it is really possible by logging in as user you use to run the tests (aab4).

This also requires re-recording all of the cassettes in this PR. And besides that, I would recommend you to change a password for your primary account, if you have used it for recording any of the cassettes below.

Reply:
Thanks for the heads-up. I'm going to look into it. No, that account is not my primary account, but I'll change the password anyway.

Reply:
Hi @tomasbedrich,

Is `gspkauth` the only cookie that needs to be kept secret? What about `__RequestVerificationToken`? Thanks.

Reply:
Fixed `gspkauth` in 8e1887fbdc76e8040e8b53edeb74f29b73233362. Still not sure about `__RequestVerificationToken`, but I built my solution in such a way that adding that cookie as well would be a one-liner.

Reply:
I am not sure about `__RequestVerificationToken`, but I think it would be better to strip it it too (preventively).

Reply:
Fixed.